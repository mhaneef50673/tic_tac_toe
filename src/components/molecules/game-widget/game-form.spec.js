import React from "react";
import { mount } from "enzyme";
import { mountToJson } from "enzyme-to-json";
import GameForm from "./game-form";

describe("GameForm Component", () => {
  it("should render the component", () => {
    const tree = mount(
      <GameForm
        playerOne="player1"
        playerTwo="player2"
        onInputChange={() => {}}
        onGameStartHandler={() => {}}
      />
    );
    expect(mountToJson(tree)).toMatchSnapshot();
  });
});
