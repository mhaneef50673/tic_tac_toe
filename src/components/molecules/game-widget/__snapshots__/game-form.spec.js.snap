// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameForm Component should render the component 1`] = `
<GameForm
  onGameStartHandler={[Function]}
  onInputChange={[Function]}
  playerOne="player1"
  playerTwo="player2"
>
  <form
    onSubmit={[Function]}
  >
    <label
      className="input-label"
    >
      Player one Name
    </label>
    <input
      className="input-text"
      onChange={[Function]}
      placeholder="Enter player one name"
      required={true}
      type="text"
      value="player1"
    />
    <label
      className="input-label"
    >
      Player two Name
    </label>
    <input
      className="input-text"
      onChange={[Function]}
      placeholder="Enter player two name"
      required={true}
      type="text"
      value="player2"
    />
    <div
      className="btn-center "
    >
      <button
        className="btn-class"
        type="submit"
      >
        Start Game
      </button>
    </div>
  </form>
</GameForm>
`;
